src/utils/findPath.js

```diff
@@ -1,5 +1,52 @@
 import core from '~/services/core';
 import web3 from '~/services/web3';
+import { PATHFINDER_HOPS_DEFAULT } from '~/utils/constants';
+
+const { ErrorCodes } = core.errors;
+const LARGE_AMOUNT = new web3.utils.BN(
+  web3.utils.toWei('1000000000000000', 'ether'),
```

Comment:
why particularly this value 1000000000000000? maybe we could add comment?

Reply:
I thought large amount as name was enough but I can add comment :)

Reply:
I will add something like "findTransitiveTransfer returns a maxFlow value when a too large amount is specified as amount. If the amount is in the order of magnitude of actual possible transfers it will return the steps of that transfer instead. We use 10¹⁵ CRC as this unrealistic amount."